c10/cuda/CUDACachingAllocator.h

```diff
@@ -334,6 +335,13 @@ inline std::shared_ptr<void> getIpcDevPtr(std::string handle) {
   return Chosen::getIpcDevPtr(handle);
 }
 
+// We allow access to the native allocator to enable fallback on the backends
+// if needed
+namespace Native {
+Allocator* get();
+void init(int device_count);
+}
```

Comment:
Should we make all `Native` methods accessible here for proper fallback if needed?